src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const totalPop = document.getElementsByClassName('total-population');
+
+const popul = Array.from(document.getElementsByClassName('population'))
```

Comment:
```suggestion
const popul = [...document.querySelectorAll('.population'))]
```
Такая запись короче и читабельней, мне кажется